### DISCUSSION TITLE
Nerf Mythic BT trash to the normal values!

### DISCUSSION THREAD

**Discussion by Krateozshael** [chromiecraft/chromiecraft#7442] [Suggestions]
I think i speak for quite alot of people, when saying: Nerf Trash. Trash is not meant to be fun or engaging, there is no reason to buff their HP, except when you want to have them bug tested. After 2-3 weeks in BT every Trash should have their respective bug reports received. 

Tuning trash up causes that alot of guilds/raidgroups are not being able to attempt a full clear of BT Mythic on one raid night. It would even out the playing field of larger guilds, that can effort raiding on multiple days vs. guilds that just have their one gaming-night a week to clear the current content, 
Its totally fine to have bosses tuned up and that you need to progress on them in a certain way, but making a raid artificially longer just due to trash tuning is unfun and unnecessary.

Thanks for everything you do dear CC Tuning Staff, but please change this for the love of "smaller" guilds.

**Comment by heyitsbench** [comment]
> After 2-3 weeks in BT every Trash should have their respective bug reports received.

Fun fact: this still isn't the case.

**Reply by Krateozshael** [reply]
Well there is a difference between "should" and "has". Not entirely my mistake, if those are not written. But if you know its not the case, that those are reported, then those bugs can be reported by you, no? :^) 

and nerfing trash doesnt mean those bugs wont exist anymore.

**Reply by elthehablo** [reply]
It is important to have multiple people checking and confirming bugs. Furthermore, it would be highly unlikely that all bugs are found within the first weeks.

Yes, some things are really obvious to spot, but keeping creatures at these high values and making players come up with new techniques to try and kill them faster will make edge-case situations and thus new bugs happen faster. We have seen this happen before.

Your last point does not make sense to me. If tuning is removed from trash (or lowered significantly) some mobs won't even get a single spell off. So yes, the bugs still exist, but the behaviour won't be observed

**Reply by veredis1** [reply]
Just reported a trash bug: https://github.com/chromiecraft/chromiecraft/issues/7452 would not saw that in normal because u just AOE pull them xD

Edit: it was a duplicate (was allready reportet, sorry for that xD) but I saw the bug I think that matters in this regard.

**Comment by elthehablo** [comment]
Our intention with mythic is to have something that is a greater challenge. The idea is to really make the raid night feel like a proper raid night where it also takes a lot of time to get to every boss.

Essentially the goal is to re-create what release day raids were like back in original TBC. This is coming from someone who did not play back then, but I have heard the stories.

This reflects back to the difficulty we had for Molten Core when it was released. We want to stick to that same kind of vibe.

**Reply by Krateozshael** [reply]
You did a good job with giving us a harder challenge on bosses. And i see that progressing on harder BOSSES is more fun and a proper challenge. Nonetheless my point is to nerf TRASH not the bosses. Trash were never part of making a raid "difficult" or not, the bosses are. 
Trash is a filler to give out some little trash loot upgrades and to make the raid not feel "empty". 

I played, raidlead and conquered MC here on CC. MC was beaten in our first month within 2 - 2 1/2hours or less. If you want to keep "the same vibe", then BT can be the same. i emphasize again, i dont want the bosses to be changed, only the "boring" filler parts between them.
Regarding that, you are FULLY ignoring the main point i make with this point. I simply want that BT and its bosses can be beaten with almost perfect execution on one raid night, since quite a bunch of guilds dont have a 2nd raidday to complete the raid on mythic. This is not a question about challenge, this is a question of making it avaiable for all guilds with proper execution to kill 9/9 mythic bosses in ONE sitting. 
As of now i dont see a way, where you can reliably do that within the usual 3 hour raid.

Additionally you would enable bigger raids/guilds to add Hyjal or other raids again to the pool of raids they can do, so you can get further bug reports on those raids, if something get messed up again (Look at sudden changes to TK / SSC (Vashj) after they were out for 1/2 year)

**Reply by elthehablo** [reply]
I did not ignore your point. The idea behind the tuning is that the mythic raids should be an activity that takes the whole night as well as potentially having to be continued later on.

When I raided on CC in a non-top tier guild back in vanilla this was normal for us when raids just released.

The idea is for it to be a steady progression.

**Reply by Krateozshael** [reply]
Also to add soemthing else: After doing half a mythic raid last week and going further this week:
Trash rn is more punishable and difficult than the majority of any boss in BT. I dont know if its the intention to make the way to a boss harder than the actual boss.

**Reply by Krateozshael** [reply]
Steady progression on bosses i fully get! 
But running out of time cuz of tuned trash hp is no steady progression, its stable non-progression. IF you cant reach the endboss at all due to tuned trash and running out of time, its no progression. its a static situation

**Comment by Krateozshael** [comment]
Adding another suggestion:
You have the majority of the community on discord. Most people dont have a github account, so they cant comment here.
Use your Discord platform to do a simple "vote" what people want, purely democratic. If the majority wants buffed hp, so shall be it.

**Comment by IskrenPlashkov** [comment]
I for one support this motion.

**Comment by B4nanaboi** [comment]
I support Krat's suggestion to open a poll on discord. My view:

The argument "trash should have enough HP to use their abilities" is 100% valid, however I think mythic trash is tuned far above that. 
The argument "now it's close to the original first weeks BT" I disagree with, because you can't compare that time back then to now. Back then, most players were in their teens/twenties and were able to form 2-4 raidnights a week. Now most people are +/- 28-40 years old, with jobs, families etc. Result: Being able to raid ONE night a week with the full core setup, which really is needed for mythic. It should be mostly hard bosses/ skills/ coordination that make you unable to finish the raid in one night, not high HP trash.

You've already polled many things and we're grateful for that, thank you. I love this server, but I think this one deserves a poll as well, not just for BT, but for mythic trash in general.

**Comment by veredis1** [comment]
I dont support that one, I like that the trash isnt just aoe them down. Its a nice change to see that trash has some kind of challenge and u can make the fights way easier if u cc some of the mobs in a group. And its mythic why should u nerf it more? Its in a kinda good state BT as a whole mythic Raid (for me the first 4 bosses do too less damage, they are basically free loot).

**Reply by B4nanaboi** [reply]
You can still have challenging trash while it has less HP.
